react-main/src/app/components/Board/index.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component, Fragment } from 'react';
+
+import Square from '../Square';
+
+class Board extends Component {
+  renderSquare(i) {
```

Comment:
We usually define component methods as arrow functions (except for react lifecycle methods). 
You can do `renderSquare = i => <Square ... />`